quickref/qsharp-quick-reference.tex

```diff
@@ -284,6 +284,59 @@
    $ \begin{bmatrix} 1 & 0 \\ 0 & e^{i\theta} \end{bmatrix} $
 }}
 
+\newbox\cnotgate
+\sbox\cnotgate{\parbox{\textwidth}{
+  \footnotesize
+   $ \begin{bmatrix} 1 & 0 & 0 & 0 \\ 0 & 1 & 0 & 0 \\ 0 & 0 & 0 & 1 \\ 0 & 0 & 1 & 0 \end{bmatrix} $
```

Comment:
If we're highlighting the sections of the matrix that differ from identity, like in Toffoli gate, could we highlight them here as well?